app/Server/src/SERVERAPI/Pages/Ranch/RanchAnimals/CreateEdit.cshtml.cs

```diff
@@ -173,6 +176,13 @@ public async Task<Command> Handle(LookupDataQuery request, CancellationToken can
                 }
                 command.CattleSubTypeOptions = new SelectList(subTypeOptions, "Id", "Value");
 
+                var prompts = _db.UserPrompts
+                    .Where(p => p.UserPromptPage == UserPromptPage.AnimalsCreateEdit.ToString() &&
+                                    p.UserJourney == UserJourney.Ranch.ToString())
+                    .ToDictionary(p => p.Name, p => p.Text);
+
+                var RanchAnimalGroupsMessage = prompts["RanchAnimalGroupsMessage"];

```

Comment:
@CodingVelocista How can I use the RanchAnimalGroupMessage data on the OnGetCreateAsync method to append the value from RanchAnimalGroupMessage  to the Title